bot.user.js

```diff
@@ -861,43 +870,36 @@ var userInterface = (function() {
                 }
             }
         },
-
-        onFrameUpdate: function() {
-            // Botstatus overlay
+        
+        onPrefChange: function() {
             var generalStyle = '<span style = "opacity: 0.35";>';
             window.botstatus_overlay.innerHTML = generalStyle + '(T / Right Click) Bot: </span>' + userInterface.handleTextColor(bot.isBotRunning);
             window.visualdebugging_overlay.innerHTML = generalStyle + '(Y) Visual debugging: </span>' + userInterface.handleTextColor(window.visualDebugging);
             window.logdebugging_overlay.innerHTML = generalStyle + '(U) Log debugging: </span>' + userInterface.handleTextColor(window.logDebugging);
             window.autorespawn_overlay.innerHTML = generalStyle + '(I) Auto respawning: </span>' + userInterface.handleTextColor(window.autoRespawn);
             window.rotateskin_overlay.innerHTML = generalStyle + '(W) Auto skin rotator: </span>' + userInterface.handleTextColor(window.rotateskin);
             window.rendermode_overlay.innerHTML = generalStyle + '(O) Mobile rendering: </span>' + userInterface.handleTextColor(window.mobileRender);
-            window.huntprey_overlay.innerHTML = generalStyle + '(P) Prey hunting: </span>' + userInterface.handleTextColor(window.huntPrey);
             window.collision_detection_overlay.innerHTML = generalStyle + '(C) Collision detection: </span>' + userInterface.handleTextColor(window.collisionDetection);
-            window.head_collision_radius_multiplier_overlay.innerHTML = generalStyle + '(A/S) Head avoidance radius multiplier: ' + window.headCollisionRadiusMultiplier + ' </span>';
-            window.body_collision_radius_multiplier_overlay.innerHTML = generalStyle + '(E/R) Body avoidance radius multiplier: ' + window.bodyCollisionRadiusMultiplier + ' </span>';
-            window.defence_overlay.innerHTML = generalStyle + '(D) Defence: </span>' + userInterface.handleTextColor(window.defence);
-            window.resetzoom_overlay.innerHTML = generalStyle + '(Z) Reset zoom </span>';
-            window.scroll_overlay.innerHTML = generalStyle + '(Mouse Wheel) Zoom in/out </span>';
-            window.quittomenu_overlay.innerHTML = generalStyle + '(Q) Quit to menu </span>';
-            window.changeskin_overlay.innerHTML = generalStyle + '(X) Change skin </span>';
-
-            window.clusteringchange_overlay.innerHTML = generalStyle + '(F) Switch clustering algorithm: ' + (bot.foodClusteringMode == 0 ? 'chancity' : 'ksofiyuk') + '</span>';
-            window.quickResp_overlay.innerHTML = generalStyle + '(ESC) Quick Respawn </span>';
+            window.collision_radius_multiplier_overlay.innerHTML = generalStyle + '(A/S) Collision radius multiplier: ' + window.collisionRadiusMultiplier + ' </span>';  
+        },
+
+        onFrameUpdate: function() {
```

Comment:
@ErmiyaEskandary is this the new eslint rule?

I think it wants us to say

    onFrameUpdate: function onFrameUpdate() {